projects/storefrontapp-e2e-cypress/cypress/helpers/quote.ts

```diff
@@ -586,6 +603,34 @@ export function checkLinkedItemInViewport(index: number) {
     'be.visible'
   );
 }
+/**
+ * Cancels the quote
+ */
+export function cancelQuoteProcess() {
+  log('Cancels the quote', cancelQuoteProcess.name);
+  cancelQuote();
+  clickOnYesBtnWithinRequestPopUp();
+}
+
+/**
+ * Clicks on "Cancel Quote" button
+ */
+function cancelQuote() {
+  log('Clicks on "Cancel Quote" button', cancelQuote.name);
+  cy.get('cx-quote-actions-by-role button.btn-secondary')
+    .click()
+    .then(() => {
+      cy.get('cx-quote-confirm-action-dialog').should('be.visible');
+    });
+}
+
+/**
+ * Visits the quote page
+ */
+export function visitQuotePage() {
```

Comment:
I would suggest to rename this method to: `gotToQuotePage()`
See for instance by textfield configurator: `goToProductDetailsPage()`